internal/oidc/const.go

```diff
@@ -0,0 +1,22 @@
+package oidc
+
+const (
+	fallbackOIDCIssuer = "https://login.example.com:8080"
```

Comment:
I think this constant should be removed. There is no good reason to have this URL in production code. We should find a better way to display the error or fallback in a better way.

Reply:
Yeah I agree, it was meant a as stopgap for theory testing I just didn't note it in a TODO.